controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1563,18 +1602,22 @@ bool master_thread::handle_non_intel_slave_join(Socket *sd,
     // TODO Assume repeater mode
     database.set_hostap_repeater_mode_flag(radio_mac, true);
     // TODO Assume no backhaul manager
-    database.set_hostap_backhaul_manager(radio_mac, false);
+    database.set_hostap_backhaul_manager(radio_mac, true);
```

Comment:
Why this change? And shouldn't the comment above be adapted as well?

Reply:
The controller requires at least one slave to be the backhaul manager slave, so it is better to set it by default *I think*.
@morantr can you comment?
I will update the //TODO

Reply:
Well setting the son_slave to backhaul manager is only matter on flows which we use beerocks proprietary messages, and as far as I know, we Multi-Ap standard does not mention which radio is the backhaul manager, so when we will fully work with the Multi-Ap flows, I think we could remove it in anyway.
For the non-intel agent I think you can remove this line completely.

Reply:
Ok, will do.